traitschema/schema.py

```diff
@@ -183,7 +200,7 @@ def to_hdf(self, filename, mode='w', compression=None,
             hfile.attrs['python_module'] = self.__class__.__module__
 
     @classmethod
-    def from_hdf(cls, filename):
+    def from_hdf(cls, filename, force_decode=False, encoding='utf-8'):
```

Comment:
Why are you adding these kwargs? They don't seem to be used.

Reply:
Wishful thinking. I was going to implement similar logic that would convert bytes-looking fields back into unicode. I can take it out for now until it is implemented, but I don't think it is necessary. Thoughts?